packages/pds-ember/app/styles/pds/components/modal/__config.scss

```diff
@@ -0,0 +1,196 @@
+///-------------------------------------------------------------------------///
+/// WARNING: Private Sass module (definitions may change at ANY time)
+///-------------------------------------------------------------------------///
+$_module: "PDS.Components.Modal";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../../theme";
+@use "../../tokens/color";
+@use "../_config" as Component;
+@use "../button/_config" as Button;
+@use "../../core/typography/_config" as Typography;
+
+$margin-block: 20px;
+// padding on the outer container (with the backdrop)
+// provides space around the dialog to avoid hitting the edge
+// of the viewport
+$paddingBlock: 20px;
+$paddingInline: 20px;
+
+$__blanket-backgroundColor: rgba(color.$ui-gray-900, 0.86);
+
+$__header-backgroundColor: var(--pds-modal-header-backgroundColor);
+$__header-borderColor: var(--pds-modal-header-borderColor);
+$__title-color: var(--pds-modal-title-color);
+$__icon-color: var(--pds-modal-icon-color);
+$__footer-backgroundColor: color.$ui-gray-050;
+
+@mixin theme($styles) {
+  /* [debug] #{$_module}@theme */
+  @include Component.theme(modal, $styles);
+}
+
+$PLAIN: (
+  header-backgroundColor: color.$ui-gray-050,
+  header-borderColor: theme.$borderColor,
+  title-color: color.$ui-gray-900,
```

Comment:
`icon-color` should come before `title-color` here.

Reply:
corrected